test/happygapi/youtube/videos_test.clj

```diff
@@ -1,4 +1,4 @@
-(ns happygapi.youtube.videos
+(ns happygapi.youtube.videos-test
```

Comment:
I fixed this namespace name. This fix is not related to the remaining commit.

Also, I didn't fix any other test, that fails with `lein test`.